thunder/executors/nvfuserex_impl.py

```diff
@@ -2241,6 +2241,8 @@ def map_inside_replacement(x: Any) -> None:
 
 def _linear_check(a: TensorProxy, b: TensorProxy, bias: TensorProxy | None) -> bool:
     enable_linear: None | bool = get_compile_option("nv_enable_linear", "Enable nvFuser linear.")
+    if enable_linear is None:
+        enable_linear = True
     if not enable_linear:
         return False
```

Comment:
Idk, this doesn't look that nice to me :(

Reply:
It was decided here https://github.com/Lightning-AI/lightning-thunder/pull/1371#discussion_r1823075638. Only one style change per PR per line.